source/localizable/_code-of-conduct.it.md

```diff
@@ -0,0 +1,67 @@
+Scopo
+-------
+
+Lo scopo principale di tutte le conferenze e user groups che aderiscono a questo Codice di Condotta è di essere il più inclusivi possibile rispetto al maggior numero di contributori, provenienti dai piu' eterogenei e variegati backgrounds possibili. Per questo, ci impegniamo a fornire un ambiente amichevole, sicuro e accogliente, indipendentemente dal sesso, orientamento sessuale, disabilità fisiche e/o mentali, etnia, stato socio-economico e preferenze religiose (o mancanza di tali).
+
+Questo Codice di Condotta delinea cosa ci aspettiamo da tutti i partecipanti delle nostre communities, cosí come le conseguenze per eventuali comportamenti inaccettabili.
+
+Invitiamo tutti/e coloro che partecipano ai nostri eventi ad aiutarci nel creare un' esperienza positiva e sicura per tutti/e.
+
+
+Cittadinanza Open [Source/Culture/Tech]
+--------------------------------------
+
+Uno degli scopi addizionali di questo Codice di Condotta è di favorire una cittadinanza open [source/culture/tech] incoraggiando i partecipanti a riconoscere e rafforzare le relazioni tra le nostre azioni e come esse influenzano la nostra community.
+
+Le communities sono lo specchio delle società nelle quali esse esistono ed agire in maniera positiva è essenziale per contrastare tutte le forme di inequalità e abusi di potere che esistono nella società in cui viviamo.
+
+
+Se vieni a conoscenza di qualcuno/a il/la cui esemplarmente si assicura che la nostra community sia accogliente, amichevole, e che attivamente incoraggia tutti i partecipanti nel contribuire al massimo, faccelo sapere!
```

Comment:
1. "Se vieni a conoscenza di qualcuno/a che in maniera esemplare si assicura..."
2. no comma before "e". It's a conjunction and in Italian we don't support the Oxford comma :p
3. "tutti/e i/le partecipanti"